core/ledger-client/src/ledger-client.ts

```diff
@@ -676,10 +676,7 @@ export class LedgerClient {
         return this.acsHelper.getCacheStats()
     }
 
-    /**
-     * @deprecated use postWithRetry instead, should be made private
-     */
-    private async post<Path extends PostEndpoint>(
+    public async post<Path extends PostEndpoint>(
```

Comment:
why undeprecate/unprivate these methods? Any reason not to use the retryable equivalents for the status check?